apps/smart-contracts/token/test/BlocklistTransferHook.test.ts

```diff
@@ -143,4 +143,57 @@ describe('BlocklistTransferHook', () => {
       expect(await blocklistTransferHook.getBlocklist()).to.eq(JUNK_ADDRESS)
     })
   })
+
+  describe('hook', () => {
+    let source: SignerWithAddress
+    let destination: SignerWithAddress
+    beforeEach(async () => {
+      await setupHookAndList()
+      await blocklistTransferHook.connect(owner).setToken(ppoToken.address)
+      source = user1
+      destination = user2
+    })
+
+    it('reverts if caller is not PPO', async () => {
+      expect(await blocklistTransferHook.getToken()).to.not.eq(user1.address)
+
+      await expect(
+        blocklistTransferHook.connect(user1).hook(source.address, destination.address, 1)
+      ).to.be.revertedWith('msg.sender != token')
+    })
+
+    it('reverts if source blocked', async () => {
```

Comment:
```suggestion
    it('reverts if sender blocked', async () => {
```
Please do for the other testcase names